.github/workflows/docs.yml

```diff
@@ -4,22 +4,41 @@ on:
     branches:
       - master
     paths:
-      - "docs/**"
-      - "backend/**"
-      - ".github/workflows/docs.yml"
+      - 'docs/**'
+      - 'backend/**'
+      - '.github/workflows/docs.yml'
   pull_request:
     branches:
       - master
     paths:
-      - "docs/**"
-      - "backend/**"
-      - ".github/workflows/docs.yml"
+      - 'docs/**'
+      - 'backend/**'
+      - '.github/workflows/docs.yml'
 
 defaults:
   run:
     working-directory: ./docs
 
 jobs:
+  security:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+      - uses: actions/setup-node@v2.1.5
+        with:
+          node-version: '14'
+      - name: Restore npm cache
+        uses: actions/cache@v2.1.4
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      - name: Install dependencies
+        run: npm ci
+      - name: Security
+        run: npm audit
+        continue-on-error: true
```

Comment:
Thanks -- can you remove this line, so that we ensure that errors aren't ignored?

Reply:
Yes I can - Please note this same configuration,  'continue-on-error: true'  is also applied to both frontend, and backend 'security:'. Do you want to update those settings too?

Reply:
Yeah, that would be good -- and could you also go ahead and run `npm audit fix` to fix the vulnerabilities so that the `npm audit` checks pass?

We had added 'continue-on-error: true' a while back because of an issue with outdated versions of lodash (which didn't have a patch released yet), but that issue should be fixed now.